symfony/templates/admin/notifications.html.twig

```diff
@@ -0,0 +1,60 @@
+{% extends 'base.html.twig' %}
+{% block title %}Notifications - {% endblock %}
+{% macro displayNotifications(notifications) %}
+    <table class="table table-striped table-hover">
+        <thead><tr><th>Time</th><th>Organization</th><th># Withdrawn Participants</th><th>Notified Emails</th></tr></thead>
```

Comment:
The column header for "# Withdrawn Participants" should be variable based on the type of notification (either Withdrawn or Deactivated)